### PR TITLE
Kvesna/webcore 1639

### PR DESCRIPTION
**Pull request by c69-addepar**

### CODE REVIEW

**Comment by c69-addepar** [comment]
`ember deploy production` fails with:

```- build failed
Error: ember-cli-test-loader/test-support/index.js: [BABEL]: Cannot find module '@babel/helper-get-function-arity'
Require stack:
- /Volumes/code/ember-json-viewer/node_modules/ember-cli-babel/node_modules/@babel/helper-function-name/lib/index.js
- /Volumes/code/ember-json-viewer/node_modules/ember-cli-babel/node_modules/@babel/helper-create-class-features-plugin/lib/index.js
- /Volumes/code/ember-json-viewer/node_modules/ember-cli-babel/node_modules/@babel/plugin-proposal-decorators/lib/index.js
```

File: package.json [hunk 1]
```diff
@@ -26,47 +26,56 @@
...
-    "release-it": "^14.2.2"
+    "qunit": "2.13.0",
+    "qunit-dom": "^2.0.0",
+    "release-it": "^14.2.2",
+    "yarn-deduplicate": "^3.1.0",
+    "webpack": "^5.69.1"
   },
   "engines": {
-    "node": "10.* || >= 12"
+    "node": ">= 16.*"
```
Comment (c69-addepar) [inline package.json:78]: looks like i need to add volta, as well

File: addon/components/entry-viewer.js [hunk 2]
```diff
@@ -1,50 +1,58 @@
-import Component from "@ember/component";
-import { computed } from "@ember/object";
+import Component from "@glimmer/component";
+// eslint-disable-next-line ember/no-computed-properties-in-native-classes
+import { action, computed } from '@ember/object';
 import layout from "../templates/components/entry-viewer";
-import { readOnly } from "@ember/object/computed";
 import { isPrimitive } from "../utils/value-types";
+import { tracked } from '@glimmer/tracking';
```
Comment (mixonic) [inline addon/components/entry-viewer.js:6]: This is not compatible with Ember 3.12, which Iverson runs.

**Review by mixonic — COMMENTED** [review]
@c69-addepar As this is a public project, please take care to make your pull requests understandable to any other contributor.

WEBCORE-1639 is an internal Addepar ticket to "Fix Ember 3.28 deprecations in `ember-json-viewer`"

I think this PR addresses deprecations which targeted Ember 4, or even updates code which would have worked find in Ember 4 but was not in the latest style.

@c69-addepar I think the correct path forward here is to stash this PR for later, and focus on adding Ember 3.28 to the ember try config at https://github.com/Addepar/ember-json-viewer/blob/master/config/ember-try.js, then make sure we get a green build. Deprecation messages are ok, we just need a green build on Ember 3.28.

After that we could look at add Ember 4 to the try file, but that isn't anything we at Addepar need at this moment.

**Comment by bantic** [comment]
I'm closing this as stale, but feel free to re-open!

**Pull request closed without merging**